src/api/app/controllers/webui/users_controller.rb

```diff
@@ -0,0 +1,56 @@
+
+class Webui::UsersController < Webui::WebuiController
+  before_action :require_login, only: [:index]
+  before_action :require_admin, only: [:index]
+
+  def index
+    respond_to do |format|
+      format.html do
+        @users = User.list unless switch_to_webui2?
```

Comment:
`unless switch_to_webui2` is always `true` now, so it needs to be removed.

Reply:
done